ChangeLog.d/windows_pragma_lib_bcrypt

```diff
@@ -0,0 +1,2 @@
+Changes
```

Comment:
The file needs to have a `.txt` extension.

Reply:
Thanks, I've added the extension.